Monika After Story/game/script-topics.rpy

```diff
@@ -14867,11 +14867,16 @@ label monika_wabi_sabi:
     m "It derives from traditional Japanese and Buddhistic philosophy, surrounding the acceptance of the temporary state of all things."
     m 3dub "It states that beyond everything, beauty is found in what's impermanent and imperfect."
     m 3wub "Like a scar on a rugged face, or a discolored hand, or even the mistakes we make."
+    m      "The imperfections can become beautiful stories that reminds us of our past..." 
+    m 3hub "...and our mistakes turn into lessons that shape us and make us stronger and more defined."
+    m 4wub "Like a sculptor revealing the magnificence inside a marble block, except that scultor is you, and your past."
     m 1tsu "Don't you find that concept rather romantic?"
     m 1hubsb "Ahaha~"
     m "..."
     m 6ekd "Anyway, [player], I hope you are not too self conscious about your looks."
     m 6dkd "After all, no matter how your face looks, or how your body looks..."
-    m 1subfb "You'll still be the most beautiful person in the whole world to me!"
-    m 4subfb "And that's my wabi-sabi promise!"
+    m 6dku "I'm sure there's a valuable story behind every scar, and every imperfection about you."
+    m 2eku "So please, [player], no matter what, don't beat yourself up about how you look."
```

Comment:
I feel beat yourself up fits better here. My thought process is that sometimes, there's not much you can do about initial feelings of inferiority, however you can tackle those feelings in more ways than one. You can learn from them and aim to be better, or you can beat yourself up over them until it takes over your life.

Telling someone to not be ashamed of how one looks is fine, one shouldn't be ashamed of how one looks. Unless, and I mean this in the nicest way, how you look is something you're currently working on changing. For instance, this can be a trans issue. Do you understand what I mean? The feeling of shame is not always something you can just wish away with a perspective change, however, what you do with that feeling, matters. Not beating yourself up over it is the first step, as that is just plain harmful.

Reply:
I get what you mean, but I think the wording is kind of awkward otherwise. If you can rephrase it then it's fine

Reply:
I may have overdone the commas again, but I don't understand how I can fix the wording. "don't beat yourself up on it"? "don't be so hard on yourself about how you look"? What do you think?

Reply:
I feel like this line should be put behind a if not persistent._mas_pm_love_yourself:. If the players likes how he looks, it doesn't make sense to have her assume he's beating himself up/ashamed about it.
Maybe something a little more neutral for the else: like
`
It doesn't matter to me what you look like [player], as long as you're confortable with who you are. I mean, I definitely wouldn't mind if you were cute, but that's not what's important to me.
`

Reply:
Interesting, I've never considered using a split like that, actually not really sure how to do it right. If someone wants to take this conversation and make it a little more fancy by adding if functions and stuff, that would be cool. Seeing as this is my first topic, I really just wanted to make it as simple and straight-forward as possible, but for quality assurance, I'd appreciate it if someone took it from there and made it MAS worthy.